data_structures/Queue_implementation.cpp

```diff
@@ -0,0 +1,54 @@
+// CPP program to implement Queue using
+// two stacks with costly deQueue()
+#include <bits/stdc++.h>
+using namespace std;
+
+struct Queue {
+	stack<int> s1, s2;
+
+	// Enqueue an item to the queue
+	void enQueue(int x)
```

Comment:
Functions should be documented explaining the use of its parameters, a brief description, and what it returns.